prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -62,26 +69,47 @@ class LoginCommand : OktaCommand(
         .choice("local", "test", "staging", "prod")
         .default("local", "local")
 
+    private val useApiKey by option(
+        "--useApiKey",
+        help = "Enable to sign in to Okta via an Api request instead of a login page"
+    ).flag(default = false)
+
+    private val forceRefreshToken by option(
+        "--force",
+        help = "Enable to do a login regardless of any current access tokens"
+    ).flag(default = false)
+
     override fun run() {
         val oktaApp = Environment.get(env).oktaApp ?: abort("No need to login in this environment")
-        val accessTokenFile = readAccessTokenFile()
-        if (accessTokenFile != null && isValidToken(oktaApp, accessTokenFile)) {
+        oktaConfig.init(oktaApp)
```

Comment:
Nit: I'm fine leaving it as it is, but this could be potentially further cleaned up by not storing/mutating state in the companion object in favor of just passing an instance of `OktaConfig` to methods that need it